internal/film/fimlrepo_mongo.go

```diff
@@ -0,0 +1,49 @@
+package film
+
+import (
+	imdb "github.com/PhamDuyKhang/littledetective/internal/pkg/crawler"
+	"github.com/PhamDuyKhang/littledetective/internal/types"
+	"github.com/globalsign/mgo"
+	"github.com/globalsign/mgo/bson"
+)
+
+type (
+	FilmMongoRepository struct {
+		ss *mgo.Session
+	}
+)
+
+func NewFilmMongoRepository(s *mgo.Session) *FilmMongoRepository {
+	return &FilmMongoRepository{
+		ss: s,
+	}
+}
+
+func (frp FilmMongoRepository) InsertFilm(f types.Film) (types.Film, error) {
+	session := frp.ss.Clone()
+	defer session.Clone()
+	f.ID = imdb.NewUUID()
+	err := session.DB("imdbfilms").C("movie").Insert(f)
```

Comment:
For strings/var has been reused in many places like these, consider to make it `const`.